### PR TITLE
LPD-1466 Web Content not shown by Asset Publisher in Live env after expiring older versions and performing a publication

### PR DESCRIPTION
**Pull request by jkappler**
https://liferay.atlassian.net/browse/LPD-1466

@lfbesada, thanks 😍

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jkappler** [comment]
@vendeltoreki, @4lejandrito, fyi

**Comment by jkappler** [comment]
ci:test:sf

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:echo

**Comment by 4lejandrito** [comment]
Thanks @jkappler!

**Comment by lfbesada** [comment]
ci:report:2522857555

**Review by lfbesada — APPROVED** [review]
It works like a charm :magic_wand: 
Thanks!

**Comment by jkappler** [comment]
ci:report:2522866417

**Comment by lfbesada** [comment]
ci:forward:force

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/exportimport/data/handler/JournalArticleStagedModelDataHandler.java [hunk 1]
```diff
@@ -1733,6 +1740,19 @@ private void _updateArticleVersions(JournalArticle article)
 		}
 	}
 
+	private boolean _updateAsset(
```
Comment (petrjung) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/exportimport/data/handler/JournalArticleStagedModelDataHandler.java:1743]: The private boolean _updateAsset( .. ) function could have better more explaining name like e.g.: _checkUpdateAssetInLatestVersion

**Pull request closed without merging**